project/templates/security/login.html.twig

```diff
@@ -13,11 +13,11 @@
     <div class="container-fluid div1">
         <div class="row">
             <div class="col-sm">
-                <img src="images/teds.png" class="img-fluid" alt="teds" style="padding: 10px">
+                <img src="{{ app.request.getSchemeAndHttpHost() }}/images/teds.png" class="img-fluid" alt="teds" style="padding: 10px">
```

Comment:
You should use this (i am not sure of the syntax) : 
`{{ asset('/images/teds.png') }}`